Marlin/src/feature/tmc_util.cpp

```diff
@@ -793,7 +793,9 @@
     #endif
     TMC_REPORT("CS actual\t",        TMC_CS_ACTUAL);
     TMC_REPORT("PWM scale",          TMC_PWM_SCALE);
-    TMC_REPORT("vsense\t",           TMC_VSENSE);
+    #if HAS_DRIVER(TMC2130) || HAS_DRIVER(TMC2224) || HAS_DRIVER(TMC2660) || HAS_DRIVER(TMC2208)
```

Comment:
This is the first reference we have to TMC2224. Are there other elements that go along with this?

Reply:
I put  `HAS_DRIVER(TMC2224)` in here just for completeness, as they are supported by the TMCStepper library. 
Sorry, I dont understand what you mean with: Are there other elements that go along with this?